ts/RailroadMap.ts

```diff
@@ -428,6 +436,22 @@ export class RailroadMap {
         return this.layerVisibility[layer];
     }
 
+    toggleLegacyRotate(): boolean {
+        this.legacyRotate = !this.legacyRotate;
+        this.refresh();
+        return this.legacyRotate;
+    }
+    isInLegacyOrientationMode(): boolean {
+        // it may be more appropriate to merge legacyRotate and isInLegacyOrientationMode
+        // into a public property
+        // DISCLAIMER I am neither a typescript nor js guy, so tell me what you think
+        return this.legacyRotate;
```

Comment:
This approach is fine.

```suggestion

    isInverted(): boolean {
        return this.inverted;
```